swatch-producer-azure/deploy/clowdapp.yaml

```diff
@@ -62,6 +62,15 @@ objects:
       - replicas: ${{KAFKA_BILLABLE_USAGE_REPLICAS}}
         partitions: ${{KAFKA_BILLABLE_USAGE_PARTITIONS}}
         topicName: platform.rhsm-subscriptions.billable-usage
+      - replicas: ${{KAFKA_BILLABLE_USAGE_REPLICAS}}
+        partitions: ${{KAFKA_BILLABLE_USAGE_PARTITIONS}}
+        topicName: platform.rhsm-subscriptions.billable-usage-hourly-aggregate
+      - replicas: ${{KAFKA_BILLABLE_USAGE_REPLICAS}}
+        partitions: ${{KAFKA_BILLABLE_USAGE_PARTITIONS}}
+        topicName: platform.rhsm-subscriptions.swatch-billable-usage-aggregator-billable-usage-store-repartition
```

Comment:
Can you please open PRs against our kafka configuration repository for these (I think using the same Jira reference is helpful)?